Yosemite/Yosemite/Actions/AppSettingsAction.swift

```diff
@@ -224,4 +224,12 @@ public enum AppSettingsAction: Action {
     case getLocalAnnouncementVisibility(announcement: LocalAnnouncement, onCompletion: (Bool) -> ())
 
     case setLocalAnnouncementDismissed(announcement: LocalAnnouncement, onCompletion: (Result<Void, Error>) -> Void)
+
+    // MARK: - Tax Rates
+
+    /// Stores the selected tax rate to be applied to orders. This is site-specific.
+    case storeSelectedTaxRateID(id: Int64, siteID: Int64)
```

Comment:
Should we change the naming to `setSelectedTaxRateID` instead? Just pointing that out for consistency with the rest of cases and because by using "store" there I initially thought about being related to the Stores 😅 perhaps "saveSelectedTaxRateID" ?

`loadSelectedTaxRateID` could also be `getSelectedTaxRateID` but no preference really, I see we use both "get" and "load" in different cases.

Reply:
Thanks for the comment! Changed in c73fa9393c102a55ed649cb96ec94a4cd79e6361

I normally don't like `set` or `get` as prefixes as they are assumed to be used for properties, but it's true that consistency might have more priority here.